presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -283,6 +284,12 @@ public void freeMemory(long bytes)
         memoryReservation.getAndAdd(-bytes);
     }
 
+    public void freeTaskContextMemory(long bytes)
```

Comment:
Lets remove this method, and just have `SharedLookupSource` call `TaskContext.freeMemory`